assets/js/shortcuts.js

```diff
@@ -144,6 +173,11 @@ function setupShortcuts(shortcutDepth = 2) {
     if (shortcutsContainer) {
       shortcutsContainer.style.display = "none";
     }
+  } else {
+    const shortcutsContainer = document.getElementById("shortcuts-container");
+    if (shortcutsContainer && shortcutsContainer.style.display === "none") {
+      shortcutsContainer.style.display = "";
+    }
```

Comment:
This `else` block might seem redundant, but it has been added to address dynamic content loading – i.e., if the content changes and headers are added after the initial moment the page is loaded, the shortcuts container needs to become visible (or be evaluated for visibility) again. We use a custom shortcode to fetch Markdown content from a raw GitHub URL downstream. This block makes it aware of such changes instead of keeping the shortcut container hidden.

Reply:
Makes sense, although probably fine to simply do:

```
if (shortcutsContainer) {
  shortcutsContainer.style.display = ""; // make shortcuts display visible, if hidden
}
```

Reply:
Done, thanks! baa71d49f3621eeb8a61968562b1eaf16ea141c0